src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/HealthCertificateValidationService.swift

```diff
@@ -62,15 +62,33 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
 	) {
 		
 		// 1. Apply technical validation
-		let expirationDate = healthCertificate.cborWebTokenHeader.expirationTime
 		
+		let isExpired: Bool
+		let stub = DCCSignatureVerifiableStub(error: nil)
```

Comment:
```suggestion
		let stub = DCCSignatureVerifyingStub(error: nil)
```
This one got renamed in another PR

Reply:
I think it makes sense to already inject the signature verifying service from the app delegate and have a common instance with the HealthCertificateService

Reply:
I added a shared instance that you just need to inject in this PR: #3248

Reply:
Done